src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -1422,3 +1422,54 @@ function random_orientation_dag(
     end
     return g2
 end
+
+"""
+    bernoulli_graph(Λ; rng = nothing, nodes_type = Int64)
+
+Given the symmetric matrix ``\\Lambda \\in [0,1]^{n \\times n}``, return a Bernoulli graph with ``n`` vertices. Each edge ``(i,j)`` exists with probability ``\\Lambda[i,j]``.
```

Comment:
I would recommend using unicode for the docstrings, so that they are pleasant to read in the REPL too